.github/workflows/ci.yaml

```
@@ -0,0 +1,26 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - "main"
+  push:
+    branches:
+      - "main"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+          cache: "yarn"
+      - run: yarn install
+      - name: Lint
+        run: yarn run lint
+      - name: Format
+        run: yarn run checkFormat
+      - name: Build
+        run: yarn run build
```

If linting fails, we should still run the fmt/build checks. Can do this by making them separate jobs, or by adding `if: always()`. I'm ok with either
```suggestion
      - name: Format
        if: always()
        run: yarn run checkFormat
      - name: Build
        if: always()
        run: yarn run build
```